### DISCUSSION TITLE
Should we document the IDs in the call

### DISCUSSION THREAD

**Discussion by ankitagarwaladobe** [ONDC-Official/ONDC-FIS-Specifications#27] [General]
Sometime it is difficult to understand the ids, who generates them, what they are used for and who owns them.
For example:
On_search response -   
"form": {
                  "id": "d097c2f5-cb8d-42fe-900e-dfecdede16fb",
}

Select
"items": [
        {
          "id": "66b7b9bad166-4a3f-ada6-ca063dc9d321",
          "xinput": {
            "form_response": {
              "status": true,
              "submission_id": "c844d5f4-29c3-4398-b594-8b4716ef5dbf"
            }
          }
        }
      ]

What's the difference between the IDs above.

**Comment by 92shreyansh** [comment]
The one in on_search is missed to be removed will update the same it was in place to uniquely identify the form which would not be required as it is tracked anywhich ways

In select the item.id is the ID of the item selected based on items parts of the catalog generated by BPP and sent as part of the on_search, the submission_id is the id the BAP received while submitting the form this is generated by BPP on form submission and when sent as part of the consicutive API it is used by BPP to link the submission to the transaction.

**Reply by ankitagarwaladobe** [reply]
Thanks, I had similar understanding as well w.r.t submission_id.

I was recommending to add those details to the spec as well so everyone has the same understanding.
